ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -47,7 +47,7 @@ base_images:
     name: release
     namespace: openshift
     tag: golang-1.20
-  postgresql-12-centos7:
+  postgresql-12-c8s:
     name: postgresql-12-centos7
     namespace: edge-infrastructure
```

Comment:
Given that this exists in our namespace does this really change anything?
Wouldn't we need to change this image elsewhere?

Reply:
I guess we'd need changing the image mapping, but this should get things working and IMO we can change this in a second stage to have tests better reflecting reality.
Also it has been more than one year that we do not use edge-infrastructure image, so it was already doing it with a slightly different image